models/visitor_model.py

```diff
@@ -7,7 +7,6 @@ class Visitor(IdentifiableEntity):
     __tablename__ = 'visitor'
 
     token = db.Column(String(60), nullable=False, unique=True)
-    #name = db.Column(String(60), nullable=False)  Not used as per latest diagram - CSJ 2022-01-26
```

Comment:
Should be good to just remove this then, no need to comment  it out.

Reply:
Was removed.